src/main/scala/com/spark3d/examples/test.scala

```diff
@@ -0,0 +1,141 @@
+/*
```

Comment:
Could you give a meaningful name for this file? `test.scala` is too vague, and something like `TestKDTree.scala` would be better.